tests/routes/test_places.py

```diff
@@ -36,6 +37,7 @@ def test_post_existing_place(client):
         "phone_number": "Some number",
         "latitude": 0.1,
         "longitude": 0.2,
+        "contacted": 0,
```

Comment:
Add tests for your new endpoint and ensure it works correctly (I mean it increase the counter)

Reply:
Ok that's done !